apps/mobile/src/store/key-store.ts

```diff
@@ -89,6 +105,26 @@ export function useKeyStore() {
         })
       );
     },
+    async createNewAccountsOfWallet(fingerprint: string, activeAccounts: number) {
+      const { bitcoinKeychains, stacksKeychains } =
+        await this.deriveNextAccountKeychainsFrom(fingerprint);
+
+      dispatch(
+        userAddsAccounts(
+          Array.from({ length: activeAccounts }, (_, i) => ({
+            account: {
+              id: makeAccountIdentifer(fingerprint, i),
+            },
+            // not sure why this is needed?
+            // don't think I need to pass in the same keychain for each account
```

Comment:
the keychain is different for every account

Reply:
Thanks @kyranjamie 👍  

I realised I have more work to do as I was hitting issues in `bitcoin-keychains.read.ts` with undefined `nativeSegwit  taproot`.